From cca0efb0477f9bb7d61b48ba270b885b29c0bb72 Mon Sep 17 00:00:00 2001
From: Thomas Hindoe Paaboel Andersen <phomes@gmail.com>
Date: Tue, 16 Sep 2014 21:59:50 +0200
Subject: [PATCH] test: silence a coverity report

We check the actual contents of the file on the line after but we
might as well also check the number of bytes read here.

Found by coverity. Fixes: CID#1237521
---
 src/test/test-fileio.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/test/test-fileio.c b/src/test/test-fileio.c
index e69706c3f8..92aa794a95 100644
--- a/src/test/test-fileio.c
+++ b/src/test/test-fileio.c
@@ -325,7 +325,7 @@ static void test_write_string_file(void) {
 
         assert_se(write_string_file(fn, "boohoo") == 0);
 
-        assert_se(read(fd, buf, sizeof(buf)));
+        assert_se(read(fd, buf, sizeof(buf)) == 7);
         assert_se(streq(buf, "boohoo\n"));
 
         unlink(fn);
